client/my-sites/posts/post-list.jsx

```diff
@@ -301,4 +301,9 @@ var Posts = React.createClass( {
 	}
 } );
 
-module.exports = PostList;
+export default connect(
+	( state ) => ( {
+		selectedSiteId: getSelectedSiteId( state ),
+		hasSites: !! getSites( state ).length
```

Comment:
I don't think it would hurt too much to have a selector for it (personally I don't fancy accessing state subtrees in `connect` calls), but i'm fine with the `isEmpty` too. Your call 😉